package-lock.json

```diff
@@ -1,6944 +1,8 @@
 {
   "name": "certificates",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
It looks like you have a different npm version. In npm v7, it uses `lockfileVersion` 2. 
Can you revert this change?
To prevent this issue in the future, we should add a note, .`nvmrc` or `engines` field in the `package.json`.

Reply:
Done